collector/pg_stat_statements.go

```diff
@@ -198,11 +238,13 @@ func (c PGStatStatementsCollector) Update(ctx context.Context, instance *instanc
 	if c.includeQueryStatement {
 		querySelect = fmt.Sprintf(pgStatStatementQuerySelect, c.statementLength)
 	}
+	databaseFilter := c.buildExcludedDatabasesClause()
+	userFilter := c.buildExcludedUsersClause()
 	statementLimit := defaultStatementLimit
 	if c.statementLimit > 0 {
 		statementLimit = fmt.Sprintf("%d", c.statementLimit)
 	}
-	query := fmt.Sprintf(queryTemplate, querySelect, statementLimit)
+	query := fmt.Sprintf(queryTemplate, querySelect, databaseFilter+userFilter, statementLimit)
```

Comment:
Why put them both in the same parameter? It makes it less obvious. I think it would be better to have separate parameters.